homeassistant/components/systemmonitor/sensor.py

```diff
@@ -27,6 +30,14 @@
 
 CONF_ARG = "arg"
 
+DOMAIN = "sensor"
+ENTITY_ID_FORMAT = DOMAIN + ".{}"
```

Comment:
I guess this isn't needed then

Reply:
Fixed. Thanks for the hint. My linters did not realize that.

Reply:
No problem :) Thanks for the PR. So the next step is config flow? 😁